cmd/install/install.go

```diff
@@ -0,0 +1,233 @@
+package install
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+	"runtime"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+var (
+	ErrKubectlPresent = errors.New("kubectl is already installed")
+	ErrGitPresent     = errors.New("git is already installed")
+	ErrArgoCDPresent  = errors.New("argocd is already present")
+	ErrKubectlFail    = errors.New("error installing kubectl")
+	ErrArgoCDFail     = errors.New("error installing argocd")
+	ErrCurlMissing    = errors.New("please install curl and set it in path")
+	kubectlPath       = "/usr/local/bin/kubectl"
```

Comment:
At least display this path when downloading, so the user knows what the command is doing. Preferably make it overridable, perhaps via command line option or env var.

Reply:
Sure, I'll do that